apps/ima/fields/index.js

```diff
@@ -241,6 +241,84 @@ module.exports = {
     mixin: 'input-text',
     className: ['govuk-input', 'govuk-!-width-two-thirds']
   },
+  'is-serious-and-irreversible': {
+    isPageHeading: true,
+    mixin: 'radio-group',
+    options: ['yes', 'no'],
+    validate: 'required'
+  },
+  'is-risk-in-country': {
+    mixin: 'radio-group',
+    options: ['yes', 'no'],
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
+    }
+  },
+  'reason-in-sih': {
+    labelClassName: 'visuallyhidden',
+    mixin: 'textarea',
+    validate: ['required', 'notUrl'],
+    attributes: [{
+      attribute: 'rows',
+      value: 6
+    }]
+  },
+  'why-not-get-protection': {
+    isPageHeading: false,
+    labelClassName: 'bold',
+    mixin: 'textarea',
+    validate: ['required', 'notUrl'],
+    attributes: [{
+      attribute: 'rows',
+      value: 6
+    }]
+  },
+  'country-1': {
+    mixin: 'select',
+    className: ['js-hidden'],
+    validate: ['required'],
+    options: [{
+      value: '',
+      label: 'fields.country-1.options.null'
+    }].concat(countries)
+  },
+  countryAddNumber: {
+    className: 'visuallyhidden',
+    labelClassName: 'visuallyhidden'
+  },
+  'country-2': {
+    mixin: 'select',
+    className: ['js-hidden'],
+    options: [{
+      value: '',
+      label: 'fields.country-2.options.null'
+    }].concat(countries)
+  },
+  'country-3': {
+    mixin: 'select',
+    className: ['js-hidden'],
+    options: [{
+      value: '',
+      label: 'fields.country-3.options.null'
+    }].concat(countries)
+  },
+  'country-4': {
+    mixin: 'select',
+    className: ['js-hidden'],
+    options: [{
+      value: '',
+      label: 'fields.country-4.options.null'
+    }].concat(countries)
+  },
+  'country-5': {
+    mixin: 'select',
+    className: ['js-hidden'],
+    options: [{
+      value: '',
+      label: 'fields.country-5.options.null'
+    }].concat(countries)
+  },
```

Comment:
Do we need these fields? It seems like only country-1 is used meaningfully in the other logic.

Reply:
It works only with this logic

Reply:
@muthrajan are you sure? These fields are not being used anywhere else. I tried removing these locally and the app still worked, so I think @jamiecarterHO might be right in saying only country-1 is being used so the others can be removed here